sw/src/service-worker/service-worker.js

```diff
@@ -124,9 +134,11 @@ export function setUpServiceWorker() {
     )
 
     // If not recording, fall through to default caching strategies
-    // SWR strategy for static assets that can't be precached
+    // SWR strategy for static assets that can't be precached.
+    // Skip in development environments
     registerRoute(
         ({ url }) =>
+            PRODUCTION_ENV &&
             urlMeetsDefaultCachingCriteria(url) &&
             fileExtensionRegexp.test(url.pathname),
         new StaleWhileRevalidate({ cacheName: 'other-assets' })
```

Comment:
So newest static assets are available ASAP in development mode, skip SWR strategy for static assets